localization/de/strings.po

```diff
@@ -2802,3 +2802,6 @@ msgstr "Für den angegebenen Barcode wurde nichts gefunden"
 
 msgid "Configure colors"
 msgstr "Farben konfigurieren"
+
+msgid "Use alternative quick actions"
```

Comment:
Translations are managed via Transifex ([README ref](https://github.com/grocy/grocy?tab=readme-ov-file#localization)), so the only file to be edited here is [`strings.pot`](https://github.com/grocy/grocy/blob/master/localization/strings.pot).